Content.Shared/Armable/ArmableSystem.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Content.Shared.Examine;
+using Content.Shared.Toggleable;
+using Content.Shared.Item.ItemToggle.Components;
```

Comment:
You need to sort your imports

```suggestion
using Content.Shared.Item.ItemToggle.Components;
using Content.Shared.Toggleable;
```